Index: app/src/main/java/com/tdtuer/eventing/app/AppNavigation.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.tdtuer.eventing.app\r\n\r\nimport androidx.compose.runtime.Composable\r\nimport androidx.navigation.NavHostController\r\nimport androidx.navigation.compose.NavHost\r\n\r\n// Định nghĩa hằng số cho các routes sử dụng trong ứng dụng =))\r\nobject Routes {\r\n    const val HOME = \"home\" // Trang chủ\r\n    const val EVENT_LIST = \"event_list\" // Danh sách sự kiện\r\n    const val EVENT_DETAIL = \"event_detail\" // Chi tiết sự kiện\r\n    const val EVENT_EDIT = \"event_edit\" // Tạo/Sửa sự kiện\r\n    const val PROFILE = \"profile\" // Hồ sơ người dùng\r\n    const val AUTH = \"auth\" // Xác thực (đăng nhập/đăng ký)\r\n}\r\n\r\n@Composable\r\nfun AppNavigation(navController: NavHostController) {\r\n//    NavHost quản lí màn hình dựa trn route, với màn hình khời đầu là HOME\r\n    NavHost(navController = navController, startDestination = Routes.HOME) {\r\n\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/tdtuer/eventing/app/AppNavigation.kt b/app/src/main/java/com/tdtuer/eventing/app/AppNavigation.kt
--- a/app/src/main/java/com/tdtuer/eventing/app/AppNavigation.kt	(revision 703d73bb53bbca38daa4f8c5e9ee60ce859963e8)
+++ b/app/src/main/java/com/tdtuer/eventing/app/AppNavigation.kt	(date 1758176744294)
@@ -5,6 +5,8 @@
 import androidx.navigation.compose.NavHost
 
 // Định nghĩa hằng số cho các routes sử dụng trong ứng dụng =))
+
+// chỉ định nghĩa routes dùng để chuyển trang
 object Routes {
     const val HOME = "home" // Trang chủ
     const val EVENT_LIST = "event_list" // Danh sách sự kiện
